package.json

```
@@ -36,13 +36,13 @@
   "license": "W3C-20150513",
   "dependencies": {
     "@tpluscode/rdf-string": "^0.2.27",
-    "@types/rdf-js": "^4.0.2",
     "array-permutation": "^0.2.0",
     "n3": "^1.16.3",
-    "rdf-js": "^4.0.2",
     "yaml": "^2.2.1"
   },
   "devDependencies": {
+    "rdf-js": "^4.0.2",
```

```suggestion
    "rdf-js": "^4.0.2",
```

rdf-js is a stub and should not appear as a dependency anywhere. It only existed to be a package for the `@types/rdf-js` package to provide types for. Note that all `@types/*` packages are type declarations for the corresponding `*` package (see https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#what-are-declaration-files-and-how-do-i-get-them for explanation). The change we want here is removing `rdf-js` and its corresponding type declaration `@types/rdf-js` in favor of using the standalone package `@rdfjs/types`.

---

I thought I have removed that... oh well.